### DISCUSSION TITLE
Edit Stored Procedure/Views from Database Navigator

### DISCUSSION THREAD

**Discussion by krizztianz** [dbeaver/dbeaver#23085] [Q&A] [answered]
Hello, I want to request a feature to add a menu/option for editing Views/Stored Procedures **directly from the Database Navigator**.
So we just need to **right click** on the View/Stored Procedure that we want to edit, then select **Edit Stored Procedure**.

Similiar like what we can do in SSMS to modify StoredProcedure
<img width="290" alt="image" src="https://github.com/dbeaver/dbeaver/assets/17529726/31e702f6-d4e6-45d2-bcbc-7707b0ec1647">

**Comment by E1izabeth** [comment, marked as answer]
You can chose `View Procedure` option in the context menu or press F4 and Change the procedure there - change parameters in the UI or change the SQL code.

![image](https://github.com/dbeaver/dbeaver/assets/28875055/562b2b89-165e-4c45-a9e1-f897eaceb273)

Does it satisfy your needs?

**Comment by krizztianz** [comment]
> You can chose `View Procedure` option in the context menu or press F4 and Change the procedure there - change parameters in the UI or change the SQL code.
> 
> ![image](https://github.com/dbeaver/dbeaver/assets/28875055/562b2b89-165e-4c45-a9e1-f897eaceb273)
> 
> Does it satisfy your needs?

Well, although this is not a direct approach but it is still acceptable.
Thank you.
